zerver/lib/events.py

```diff
@@ -304,6 +307,7 @@ def fetch_initial_state_data(user_profile: UserProfile,
 
     if want('user_status'):
         state['away_user_ids'] = sorted(list(get_away_user_ids(realm_id=realm.id)))
+        state['user_status_info'] = get_user_info_dict(realm_id=realm.id)
```

Comment:
I have been imagining we'd return list of dicts of this form:
```
{
    user_id: 15,
    status: "away",
    message: "out to lunch"
}
```

Rather than having 2 independent data structures, each with their own database query.  I think that model is cleaner for e.g. model (fewer things to parse) and saves a database query on the backend.

(I guess we can keep `away_user_ids` until we migrate the webapp to use it).